ranger-plugin/conf/ranger-servicedef-hawq.json

```diff
@@ -244,7 +244,7 @@
       "name": "authentication",
       "type": "enum",
       "subType": "authType",
-      "mandatory": false,
+      "mandatory": true,
```

Comment:
are we doing this lookup-side Kerberos support in case where HAWQ authentication is set to Kerberos in pg_hba.conf ? This is not supported by default when we kerberize the cluster, so usually when the whole cluster (including HAWQ) is secured, password based connection to HAWQ will still work and kerberos-based connection will not be setup by default.